test/functional/basic-functionality.spec.ts

```diff
@@ -1680,5 +1680,47 @@ describe("basic functionality", () => {
         classToClassFromExistUser.should.be.eql([fromExistUserLike1, fromExistUserLike2]);
 
     });
+    
+    it("should expose properties with json view", () => {
+        defaultMetadataStorage.clear();
+
+        @Exclude()
+        class User {
+
+            id: number;
+
+            @Expose()
+            firstName: string;
+
+            @Expose()
+            lastName: string;
+
+            password: string;
+        }
+
+        const user = new User();
+        user.firstName = "Umed";
+        user.lastName = "Khudoiberdiev";
+        user.password = "imnosuperman";
+
+        const plainUser = {
+            firstName: "Umed",
+            lastName: "Khudoiberdiev"
+        };
+
```

Comment:
better to move `user` and `plainUser` block into controller action code to avoid extra thinking about their scope and why they are defined here.